### DISCUSSION TITLE
Tailwind gave me a reason to try with design and made web dev exciting

### DISCUSSION THREAD

**Discussion by benlatz** [tailwindlabs/tailwindcss#3215] [Kind words]
When I first started learning web development and working on my first product, I was using Bootstrap, because I thought that's just the way of the world. And even though I've been good a graphic design for years before then, it felt like a lost cause to even try doing custom design. Bootstrap was just too much structure to break out of without significant effort. And no way I was going to do custom CSS all the way through.

Then I found Tailwind and realized that I could reproduce pretty much any design nearly perfectly writing no custom CSS. It gave me a reason to start designing custom interfaces for web, as it was easy and enjoyable to translate that into real code. So easy, that I began skipping the mockup phase and was able to design directly in HTML. This is so productive that I can't imagine using anything else for CSS — this feels like the way things should be. Now I can be proud of my web applications and be excited to build instead of tolerating Bootstrap restrictions.

On top of the experience of using it, I admire Adam and Steve's journey of growing Tailwind and am always impressed with their output, in quality, attention to detail, presentation/experience, and how lean the team is. (h/t to Steve too for improving my design as well)

Thanks for what you guys do and it's even cooler that you can make an amazing business out of open source - everyone wins.

**Comment by simonswiss** [comment]
This is amazing @benlatz ❤️

Designing in HTML is truly empowering with Tailwind CSS, eh?
